contracts/extensions/CoinMachine.sol

```diff
@@ -61,10 +61,19 @@ contract CoinMachine is ColonyExtension {
 
   bool evolvePrice; // If we should evolve the price or not
 
+  mapping(address => uint256) metatransactionNonces;
```

Comment:
Can we not roll these into `ColonyExtension` as a base class? Is it because it'll mess up the storage slots?

Reply:
Yep.